AlfrescoApp/View Controllers/FileFolder View Controller/FileFolderCollectionViewController.m

```diff
@@ -583,7 +583,7 @@ - (void)collectionView:(UICollectionView *)collectionView didDeselectItemAtIndex
 {
     if (self.isEditing)
     {
-        AlfrescoNode *selectedNode = [self.dataSource alfrescoNodeAtIndex:indexPath.item];
+        AlfrescoNode *selectedNode = [self.searchDataSource alfrescoNodeAtIndex:indexPath.item];
```

Comment:
I think the data source object used here should depend on wether the search results are displayed or not, just like above.